frontend/guardian.rb

```diff
@@ -80,9 +72,18 @@ def rejected_beep(gpio_command)
 DoorAuthorization.auto_upgrade!
 TagLog.auto_upgrade!
 
-close_door(gpio_command)
+close_door(gpio_command, :front_door)
 `#{gpio_command} -g mode 9 out`
-close_door(gpio_command)
+close_door(gpio_command, :front_door)
+
+close_door(gpio_command, :rollup_door)
```

Comment:
I know this duplication pattern where the door close gpio command is invoked once, the mode is set and the same gpio command is invoked again was in the existing code.  But do you know why?  I am not sure the first invocation is necessary.  @raindrift, do you know?